src/test/java/oop/DumpTruckTest.java

```diff
@@ -0,0 +1,30 @@
+package oop;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import oop.automobiles.DumpTruck;
+import oop.automobiles.Manual;
+
+public class DumpTruckTest {
+
+    @Test
+    public void testTruck() {
+        DumpTruck myDumpTruck = new DumpTruck("NYCJAVAGIRL", 8);
+        
+        assertEquals("NYCJAVAGIRL", myDumpTruck.getLicense());
+
+        assertEquals(8, myDumpTruck.getWheels());
+
+        myDumpTruck.loadGarbage("Trash");
+        assertEquals(1, myDumpTruck.getCargo().size());
+
+        myDumpTruck.dumpGarbage(null);
```

Comment:
be careful with methods that pass null - the ideal solution here would be to remove the String argument from the dumpGarbage() implementation in the DumpTruck class, rather than passing null.